query/src/storages/fuse/constants.rs

```diff
@@ -27,3 +30,16 @@ pub const FUSE_TBL_SNAPSHOT_PREFIX: &str = "_ss";
 pub const DEFAULT_BLOCK_PER_SEGMENT: usize = 1000;
 pub const DEFAULT_ROW_PER_BLOCK: usize = 1000 * 1000;
 pub const DEFAULT_BLOCK_SIZE_IN_MEM_SIZE_THRESHOLD: usize = 100 * 1024 * 1024;
+
+lazy_static! {
+    pub static ref RESERVED_OPTS: HashSet<&'static str> = {
```

Comment:
Looks better move to sql crate? Let's keep sql crate not depends on the storage.

Reply:
> Looks better move to sql crate? Let's keep sql crate not depends on the storage.

OK